frontend/app-development/layout/AppBar/AppBar.tsx

```diff
@@ -64,17 +74,36 @@ export const AppBar = ({ activeSubHeaderSelection, showSubMenu }: IAppBarProps)
         </ul>
         <div className={classes.rightContent}>
           <div className={classes.rightContentButtons}>
-            {/* TODO: Enable cypress usecase test when below button is enabled in prod/dev (testing/cypress/src/integration/usecase/usecase.js:57) */}
-            {!_useIsProdHack() && (
-              <Button
-                className={classes.previewButton}
-                onClick={handlePreviewClick}
-                variant={ButtonVariant.Outline}
-                data-testid={TopBarMenu.Preview}
+            <Button
+              className={classes.previewButton}
+              onClick={handlePreviewClick}
+              variant={ButtonVariant.Outline}
+              data-testid={TopBarMenu.Preview}
+            >
+              {t('top_menu.preview')}
+              <div
```

Comment:
Bruk heller `<span>` her. `<div>` er et blokkelement og skal ikke brukes inni knapper, som er inline. Det samme gjelder inni popoveren. Du kan bruke `display: inline-block` i CSS for å få den til å oppføre seg som et blokkelement hvis det er nødvendig.